docs/python/settings-reference.md

```diff
@@ -23,6 +23,7 @@ Refer to [User and workspace settings](/docs/getstarted/settings.md) to find our
 | python.envFile | `"${workspaceFolder}/.env"` | Absolute path to a file containing environment variable definitions. |
 | python.terminal.launchArgs | `[]` | Launch arguments given the Python interpreter when running a file. |
 | python.terminal.executeInFileDir | `false` | Indicates whether to run a file in the file's directory instead of the current folder. |
+| python.terminal.activateEnvironments | `true` | Indicates whether the **Python: Select Interpreter** command allows selection of virtual environments. |
```

Comment:
"Indicates whether any selected virtual or conda environment should be activated first when using the `Create Terminal` or any other operation involving the terminal (e.g. `Send Python File to Terminal`)."